packages/app-parachains/src/Overview/Register.tsx

```diff
@@ -0,0 +1,157 @@
+// Copyright 2017-2020 @polkadot/app-democracy authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { ParaInfo } from '@polkadot/types/interfaces';
+import { SubmittableExtrinsic } from '@polkadot/api/promise/types';
+
+import BN from 'bn.js';
+import React, { useMemo, useRef } from 'react';
+import { useApi, useFormField, useModal } from '@polkadot/react-hooks';
+import { createType } from '@polkadot/types';
+import { u8aToHex, u8aToString } from '@polkadot/util';
+
+import { Button, Dropdown, InputFile, InputNumber, Modal, TxButton } from '@polkadot/react-components';
+
+import { useTranslation } from '../translate';
+
+interface Props {
+  nextFreeId?: BN;
+  sudoKey: string;
+}
+
+type Scheduling = 'Always' | 'Dynamic';
+
+const schedulingOptions = [{
+  text: 'Always',
+  value: 'Always'
+}, {
+  text: 'Dynamic',
+  value: 'Dynamic'
+}];
+
+function isWasmValid (wasm: Uint8Array): boolean {
+  return wasm.subarray(0, 4).toString() === '0,97,115,109'; // '\0asm'
+}
+
+export default function Register ({ nextFreeId = new BN(1000), sudoKey }: Props): React.ReactElement<Props> | null {
```

Comment:
```suggestion
const DEFAULT_FREE = new BN(1000);

export default function Register ({ nextFreeId = NEXT_FREE, sudoKey }: Props): React.ReactElement<Props> | null {
```

The alternative is that a new object would be created on each render.